src/components/PageTransition/index.jsx

```diff
@@ -1,33 +1,85 @@
 import React, { Component } from 'react';
-import MatButton from '@material-ui/core/Button';
 import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import Typography from '@material-ui/core/Typography';
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon';
+import ArrowLeftIcon from 'mdi-react/ArrowLeftIcon';
+import Button from '../Button';
+import { THEME } from '../../utils/constants';
 
 @withStyles(theme => ({
   button: {
     margin: theme.spacing.unit,
   },
-  leftIcon: {
-    marginRight: theme.spacing.unit,
+  iconSmall: {
+    fontSize: 20,
+  },
+  smallText: {
+    fontSize: '10px',
+    color: fade(THEME.PRIMARY_TEXT_LIGHT, 0.5),
+  },
+  largeText: {
+    color: fade(THEME.PRIMARY_TEXT_LIGHT, 0.8),
+    fontSize: '14px',
+  },
+  rowFlex: {
+    display: 'flex',
+    '& svg': {
+      transition: theme.transitions.create('fill'),
+      fill: fade(THEME.PRIMARY_TEXT_LIGHT, 0.5),
+    },
   },
   rightIcon: {
     marginLeft: theme.spacing.unit,
+    marginTop: 'auto',
+    marginBottom: 'auto',
   },
-  iconSmall: {
-    fontSize: 20,
+  rightButtonText: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    textAlign: 'left',
+  },
+  leftIcon: {
+    marginRight: theme.spacing.unit,
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+  leftButtonText: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    textAlign: 'right',
   },
 }))
 export default class PageTransition extends Component {
   render() {
-    const { classes } = this.props;
+    const { classes, variant } = this.props;
 
     return (
-      <MatButton
+      <Button
         {...this.props}
         variant="contained"
         color="default"
         className={classes.button}>
-        {this.props.children}
-      </MatButton>
+        <div className={classes.rowFlex}>
+          {variant === 'prev' && <ArrowLeftIcon className={classes.leftIcon} />}
+          <div
+            className={
+              variant === 'prev'
+                ? classes.leftButtonText
+                : classes.rightButtonText
+            }>
+            <Typography className={classes.smallText}>{variant}</Typography>
```

Comment:
The Typography element has a `variant` prop we could probably use. I would prefer using that instead of using a custom class `smallText` and `largeText`. Give it a try and see if it works.

https://material-ui.com/api/typography/#typography-api